caas/kubernetes/provider/k8s_test.go

```diff
@@ -279,10 +490,10 @@ func getBasicPodspec() *specs.PodSpec {
 		WorkingDir:   "/path/to/here",
 		Config: map[string]interface{}{
 			"foo":        "bar",
-			"restricted": "'yes'",
+			"restricted": "yes",
 			"bar":        true,
-			"switch":     "on",
-			"brackets":   `'["hello", "world"]'`,
+			"switch":     true,
```

Comment:
why this change?

Reply:
because no need to do 2nd round of yaml unmarshal anymore.